cards.js

```diff
@@ -16,8 +16,45 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
-
+function buildDeck(values, suits){
 // Put some code in here!
+deck =[]
+for(var v = 0; v < values.length; v++){
+  for(var s = 0; s < suits.length; s++){
+    val = values[v];
+    suit = suits[s];
+      var card = {
+        cardface: val + " of " + suit,
```

Comment:
Storing all these properties on the objects instead of calculating them at runtime is great design. :+1: